proto/regen/ecocredit/basket/v1/query.proto

```diff
@@ -98,4 +115,48 @@ message QueryBasketBalanceRequest {
 message QueryBasketBalanceResponse {
   // balance is the amount of the queried credit batch in the basket.
   string balance = 1;
-}
\ No newline at end of file
+}
+
+// BasketInfo is the human-readable basket information.
+message BasketInfo {
+  // basket_denom is the basket bank denom.
+  string basket_denom = 1;
+
+  // name is the unique name of the basket specified in MsgCreate. Basket
+  // names must be unique across all credit types and choices of exponent
+  // above and beyond the uniqueness constraint on basket_denom.
+  string name = 2;
+
+  // disable_auto_retire indicates whether or not the credits will be retired
+  // upon withdraw from the basket.
+  bool disable_auto_retire = 3;
+
+  // credit_type_abbrev is the abbreviation of the credit type this basket is
+  // able to hold.
+  string credit_type_abbrev = 4;
+
+  // date_criteria is the date criteria for batches admitted to the basket.
+  DateCriteria date_criteria = 5;
+
+  // exponent is the exponent for converting credits to/from basket tokens.
+  uint32 exponent = 6;
+
+  // curator is the address of the basket curator who is able to change certain
+  // basket settings.
+  string curator = 7;
+}
+// BasketBalanceInfo is the human-readable basket balance information.
+message BasketBalanceInfo {
+  // batch_denom is the denom of the credit batch
+  string batch_denom = 1;
+
+  // balance is the amount of ecocredits held in the basket
+  string balance = 2;
+
+  // basket_denom is the basket bank denom.
+  string basket_denom = 3;
+
+  // batch_start_date is the start date of the batch. This field is used
+  // to create an index which is used to remove the oldest credits first.
+  google.protobuf.Timestamp batch_start_date = 4;
+}
```

Comment:
should we leave some of these fields out? `batch_start_date` was only in the table so it could be used as an index, and the `basket_denom` seems a bit redundant since its in the request already.